### PR TITLE
fix(docs): nvidia store changes

### PR DESCRIPTION
**Pull request by YiIdirim**
### Description

- Minor changes to the docs for better clarity.
- Fixed hyperlink not working

If you could merge this @jef ASAP that would be great.

### CODE REVIEW

File: docs/reference/filter.md [hunk 1]
```diff
@@ -234,18 +234,18 @@ Used with the `SHOW_ONLY_SERIES` variable.
...
-| Ireland | `✔` | `✔` | `✔` | `✔`  | `nvidia-gb` | Scan |
-| Luxembourg | `✔` | `✔` | `✔` | `✔`  | `nvidia-fr` | LDLC |
-| Spain | `✔` | `✔` | `✔` | `✔`  | `nvidia-es` | LDLC |
-| United Kingdom | `✔` | `✔` | `✔` | `✔`  | `nvidia-gb` | Scan |
-| USA | `✔` | `✔` | `✔` | `✔`  | `bestbuy` | BestBuy |
-
+    Use the Nvidia Store to check stock for Founders Edition cards only (3060 Ti/3070/3080/3090). For USA use `bestbuy` instead.
+
+???+ info
+    Use the `store` variable for your chosen country. For example, to check for a FE card in Belgium you would use the `nvidia-fr` store.
```
Comment (jef) [inline docs/reference/filter.md:240]:
```suggestion
    Use the `STORE` variable for your chosen country. For example, to check for a FE card in Belgium you would use the `nvidia-fr` store.
```

**Review by jef — COMMENTED** [review]
This make sense! Thanks for the update. I'll merge after the suggestion change if you don't mind ☺️

**Comment by YiIdirim** [comment]
Done.

**Comment by YiIdirim** [comment]
> This make sense! Thanks for the update. I'll merge after the suggestion change if you don't mind ☺️ 

Done! And I sneaked in a very minor addition to the Nvidia-GB store as I believe the buy button text has changed.

**Review by jef — APPROVED** [review]
Looks good! Thanks ☺️ 🚀

**Pull request merged**